src/vis/vis-view.js

```diff
@@ -49,113 +30,17 @@ var Vis = View.extend({
-    var apiKey = options.apiKey;
-    var datasource = vizjson.datasource;
-    // TODO: We can use something else to differentiate types of "datasource"s
-    var isNamedMap = !!datasource.template_name;
-
-    if (isNamedMap) {
-      layerGroupModel = new CartoDBLayerGroupNamedMap({
-        apiKey: apiKey
-      }, {
-        layersCollection: this._layersCollection
-      });
-    } else {
-      layerGroupModel = new CartoDBLayerGroupAnonymousMap({
-        apiKey: apiKey
-      }, {
-        layersCollection: this._layersCollection
-      });
-    }
-
-    if (isNamedMap) {
-      endpoint = [WindshaftConfig.MAPS_API_BASE_URL, 'named', datasource.template_name].join('/');
-      WindshaftMapClass = WindshaftNamedMap;
-    } else {
-      endpoint = WindshaftConfig.MAPS_API_BASE_URL;
-      WindshaftMapClass = WindshaftAnonymousMap;
-    }
-
-    var windshaftClient = new WindshaftClient({
-      endpoint: endpoint,
-      urlTemplate: datasource.maps_api_template,
-      userName: datasource.user_name,
-      forceCors: datasource.force_cors || true
-    });
-
-    var modelUpdater = new ModelUpdater({
-      layerGroupModel: layerGroupModel,
-      dataviewsCollection: this._dataviewsCollection,
-      layersCollection: this._layersCollection,
-      analysisCollection: this._analysisCollection
-    });
-
-    // Create the WindshaftMap
-    this._windshaftMap = new WindshaftMapClass({
-      apiKey: apiKey,
-      statTag: datasource.stat_tag
-    }, {
-      client: windshaftClient,
-      modelUpdater: modelUpdater,
-      dataviewsCollection: this._dataviewsCollection,
-      layersCollection: this._layersCollection,
-      analysisCollection: this._analysisCollection
-    });
-
-    // Create the Map
-
-    var allowDragging = util.isMobileDevice() || vizjson.hasZoomOverlay() || vizjson.scrollwheel;
-
-    var mapConfig = {
-      title: vizjson.title,
-      description: vizjson.description,
-      maxZoom: vizjson.maxZoom,
-      minZoom: vizjson.minZoom,
-      legends: vizjson.legends,
-      bounds: vizjson.bounds,
-      center: vizjson.center,
-      zoom: vizjson.zoom,
-      scrollwheel: !!this.scrollwheel,
-      drag: allowDragging,
-      provider: vizjson.map_provider,
-      vector: vizjson.vector
-    };
-
-    this.map = new Map(mapConfig, {
-      layersCollection: this._layersCollection,
-      windshaftMap: this._windshaftMap,
-      dataviewsCollection: this._dataviewsCollection
-    });
-
-    // If a CartoDB embed map is hidden by default, its
-    // height is 0 and it will need to recalculate its size
-    // and re-center again.
-    // We will wait until it is resized and then apply
-    // the center provided in the parameters and the
-    // correct size.
-    var map_h = this.$el.outerHeight();
-    if (map_h === 0) {
-      $(window).bind('resize', function () {
-        this._onResize();
-      }.bind(this));
-    }
+  render: function () {
+    // TODO: Get this from this.model
```

Comment:
What's "this"? I'd recommend to create an issue to keep track of work your plan to do in the future instead of leaving a TODO here